packages/ui/v2/core/apps/AllApps.tsx

```diff
@@ -61,11 +62,13 @@ function CategoryTab({ selectedCategory, categories }: CategoryTabProps) {
   return (
     <div className="relative mb-4 flex flex-col justify-between lg:flex-row lg:items-center">
       <h2 className="text-lg font-semibold text-gray-900 ">
-        {t("explore_apps", {
-          category:
-            (selectedCategory && selectedCategory[0].toUpperCase() + selectedCategory.slice(1)) ||
-            t("all_apps"),
-        })}
+        {searchText
```

Comment:
Changing the title of the `AllApps` component to "Search" when there is a app search happening